libs/core-ui/src/lib/util/getFeatureOptions.ts

```diff
@@ -14,14 +14,17 @@ export function getFeatureOptions(
       const key = JointDataset.DataLabelRoot + index.toString();
       const meta = jointDataset.metaDict[key];
       const options = meta.isCategorical
-        ? meta.sortedCategoricalValues?.map(
-            (optionText: string | number, index: number) => {
+        ? meta.sortedCategoricalValues
+            ?.filter(
+              (optionText: string | number, _: number) =>
+                optionText !== undefined && optionText !== null
```

Comment:
I think lint would not allow null to pass

Reply:
it seems to be passing, at least I saw null checks elsewhere too

Reply:
We definitely don't want null in our repo. it is very dangerous to mix null and undefined.
do you mind change to !!optionText